src/main/java/bio/terra/service/filedata/azure/AzureSynapsePdao.java

```diff
@@ -1050,24 +1072,99 @@ public int getTableTotalRowCount(
   }
 
   public ColumnStatisticsDoubleModel getStatsForDoubleColumn(
-      Column column, String dataSourceName, String parquetFileLocation) {
-    throw new FeatureNotImplementedException(
-        "This feature is not yet supported for Azure-backed datasets.");
-    //    return new ColumnStatisticsNumericModel();
+      Column column, String dataSourceName, String parquetFileLocation, String filter) {
+    final String sql =
+        queryColumnStats(
+            column, dataSourceName, parquetFileLocation, filter, queryNumericColumnStatsTemplate);
+    ColumnStatisticsDoubleModel doubleModel =
+        (ColumnStatisticsDoubleModel)
```

Comment:
It complains without it and I haven't figured out a better way. I think it's complaining because the "dataType" field belongs to just the parent `ColumnStatisticsModel`. 
Maybe a good chance to bug @pshapiro4broad 😄

Reply:
Is there a way to do that when the model is defined by the open API spec?

Reply:
This problem is a covariant return type issue; there's probably a way to solve this using a custom template. There might be an option for this although I couldn't find it and could only find one open issue that's slightly related regarding covariant return types and `allOf`. https://github.com/OpenAPITools/openapi-generator/issues/1904

I would fix this by splitting it up into two operations. You could use the fluent API here too since it's a setter but I think the non-fluent version reads better.
```
    ColumnStatisticsDoubleModel doubleModel = new ColumnStatisticsDoubleModel();
    doubleModel.setDataType(column.getType().toString());
```

Reply:
Thanks for looking into this, @pshapiro4broad ! I think that's a straightforward workaround.